tools/Makefile.unix

```diff
@@ -522,6 +522,11 @@ ifeq ($(CONFIG_RAW_BINARY),y)
 	$(Q) $(OBJCOPY) $(OBJCOPYARGS) -O binary $(BIN) nuttx.bin
 	$(Q) echo nuttx.bin >> nuttx.manifest
 endif
+ifeq ($(CONFIG_MPFS_OPENSBI),y)
+	@echo "CP: nuttx.sbi"
+	$(Q) $(OBJCOPY) -O binary -j .text.sbi -j .ddrstorage $(BIN) nuttx.sbi
```

Comment:
Please add this file to the nuttx.manifest file as is done above that way we get the Artifact picked up as part of the ci results which makes testing on hardware easier.

Reply:
Ok!

Reply:
Also this change may also be needed in the win top level makefile.

Reply:
Sure!